docs/examples/test-app/deploy.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Get a Kubernetes cluster ready before deploying the application
+
+# Deploy the Application CRD with the controller on the Kubernetes cluster
+make deploy
+
+# Install the test CRD to the cluster
+kubectl apply -f test_crd.yaml
+
+# Create the application object
+kubectl apply -f application.yaml
+
+# Verify application's status/components field shows all components
+# Verify ownerReferences is added to all components' spec
```

Comment:
Does the script wait by itself ?
Do we need a readln OR a sleep here ?

Reply:
We validate the components' specs and application's status manually, but I can add a readln in the script just in case we may want to automate the script later.